src/resources/views/crud/buttons/quick.blade.php

```diff
@@ -23,6 +23,25 @@
         $wrapper['href'] = ($wrapper['href'])($entry, $crud);
     }
     $wrapper['class'] = $wrapper['class'] ?? $defaultClass;
+    //if ajax enabled
+    $ajax_enabled = $button->meta['ajax'] ?? false;
+    if($ajax_enabled) {
+        $wrapper['data-route'] = $wrapper['href'];
+		$wrapper['data-method'] = $button->meta['ajax']['method'] ?? 'POST';
+        $wrapper['data-refresh-table'] = $button->meta['ajax']['removesEntryFromTable'] ?? false;
```

Comment:
`removeEntryFromTable` is not what this does. 
Sorry if I didn't make myself clear when I explained my expectations for a table refresh. 

I think there are 2 scenarios:
1 - you do something with the entry but don't remove it (Eg, changing the status)
2 - you do remove the entry (mark it as deleted, or actually delete it). 

I think for starters we should cater only to **1**. `data-refresh-table => refreshCrudTable ?? false`. 

We can introduce separately another configuration (**but I think we shouldn't do it now, only if we need or people ask for it**), like `removesEntryFromTable`. If developer enables this, we need to do the `is there only one item in the table ?` verification that we do on delete and trash. 
But let's not work on that now.

```suggestion
        $wrapper['data-refresh-table'] = $button->meta['ajax']['refreshCrudTable'] ?? false;
```

Reply:
I was expecting this. I intentionally named it `data-refresh-table`. 😬.

Done!